src/main/tools/checkstyle/checkstyle-suppressions.xml

```diff
@@ -6,4 +6,5 @@
 <suppressions>
   <suppress files="src[\\/]main[\\/]generated[\\/]" checks="."/>
   <suppress files="target[\\/]" checks="."/>
+  <suppress files=".mvn[\\/]" checks="."/>
```

Comment:
I won't personally be using mvnw, but I have no preference on whether to add it or not

Reply:
Let's not then. I'm not a huge fan of it.

Reply:
It doesn't hurt to use this suppressions file as is though.